mystery_o_matic/output/latex/__init__.py

```
@@ -0,0 +1,148 @@
+        args["weaponsTable"] = generate_latex_weapons_table(len(weapons_options))
+
+        for (i, char) in enumerate(mystery.get_characters()):
+            args[f"CHAR{i+1}"] = names_html[f"CHAR{i+1}"]
+
+        for room, name in locations.names[language].items():
+            if room not in locations.rindices:
+                continue # skip any missing place
+            index = locations.rindices[room]
+            args[index + "REP"] = get_emoji_name(locations.representations[index])
+
+        for l, w in locations.weapon_locations.items():
```

Ruff

Ambiguous variable name: `l`

(E741)